v3/src/components/case-table/column-header.tsx

```diff
@@ -12,17 +12,19 @@ import { kIndexColumnKey, TRenderHeaderCellProps } from "./case-table-types"
 import { ColumnHeaderDivider } from "./column-header-divider"
 import { useRdgCellFocus } from "./use-rdg-cell-focus"
 
-export function ColumnHeader({ column }: Pick<TRenderHeaderCellProps, "column">) {
+export function ColumnHeader({ column, attrIdToEdit, setAttrIdToEdit }: TRenderHeaderCellProps) {
```

Comment:
As mentioned elsewhere, my inclination would be to store this information in the `CollectionTableModel` rather than in React state and passed as props.